src/org/zaproxy/zap/extension/ascanrules/TestSQLInjection.java

```diff
@@ -779,6 +779,98 @@ public void scan(HttpMessage msg, String param, String origParamValue) {
 							return;
 						}
 					}
+					// Check 4a
+					// this check is to see whwter the "1" will give the same result as "1*2/2"
+
+					if (!sqlInjectionFoundForUrl && countExpressionBasedRequests < doExpressionMaxRequests){
+						//get a value 2 times bigger
+						int paramtwice = paramAsInt * 2;
+						String modifiedParamValue4a = String.valueOf(paramtwice) + "/2";
+
+						//and prepare a request to set the parameter value to a string value like "2/2", if the original parameter value was "1"
+						//those of you still paying attention will note that if handled as expressions (such as by a database), these represent the same value.
```

Comment:
OMG ignore me....at 01:35 it seems I'm having a brain fart or something.....sigh